### PR TITLE
Adds a new solo fugitive backstory: The Invisible Man

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

A disgruntled scientist, formerly the project lead at an experimental cloaking technology lab, now on the run for stealing the very prototypes they helped create. 

image

You start off with a toggleable cloaking implant (makes you a bit more invisible than a cloaked ninja), and some Stabilized Saturn-X injectors (Which turn you COMPLETELY invisible). 

The three Saturn-X injectors grant you a cumulative 30 minutes of total invisibility (with the caveat and associated complications of you having to be naked and not able to just take them all at once consequence-free). It may not be as powerful as Waldo and his almighty Knock spell, but don't underestimate how much the camouflage can do for you in the dark. 

image

So go on, blend in with the crew. When the hunters come looking, shed your uniform and disappear...

The color of this PR is: **Purple**.
## Why It's Good For The Game

Adds some variety to solo fugitive selections (Waldo is still a bastard). Gives the opportunity for High-End SS13 Stealth Gameplay.
## Changelog
:cl: Rhials
add: New solo fugitive backstory: The Invisible Man!
spriteadd: New purple medipen and admittedly bad camouflage icon hud button sprite.
/:cl:

### CODE REVIEW

**Comment by VladinXXV** [comment]
hey cool pr but can we get some screenshots that actually include this so-called invisible man? all you've posted are the pens laying on the ground and an empty maints shaft.

**Comment by tralezab** [comment]
super cool fugitive, and a good reminder that I should really datumize them

File: code/modules/antagonists/fugitive/fugitive_outfits.dm [hunk 1]
```diff
@@ -248,3 +248,22 @@
...
+	uniform = /obj/item/clothing/under/suit/black_really
+	back = /obj/item/storage/backpack/satchel/leather
+	shoes = /obj/item/clothing/shoes/laceup
+	glasses = /obj/item/clothing/glasses/monocle
+	mask = /obj/item/clothing/mask/cigarette/pipe
+	ears = /obj/item/radio/headset
+
+	backpack_contents = list(
+		/obj/item/reagent_containers/hypospray/medipen/invisibility = 3
+		)
```
Comment (san7890) [inline code/modules/antagonists/fugitive/fugitive_outfits.dm:263]:
```suggestion
	backpack_contents = list(
		/obj/item/reagent_containers/hypospray/medipen/invisibility = 3,
	)
```

lets do it right if we're going to multiline

File: code/modules/antagonists/fugitive/fugitive_equipment.dm [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/obj/item/implant/camouflage
+	name = "experimental camouflage implant"
+	desc = "Allows its owner to blend in with their surroundings. Cool!"
+	actions_types = list(/datum/action/item_action/camouflage)
+
+/obj/item/implant/camouflage/emp_act(severity)
+	. = ..()
+
+	if(prob(15 * severity))
```
Comment (san7890) [inline code/modules/antagonists/fugitive/fugitive_equipment.dm:9]: should we have a `min()` here with some kind of default value (created as a define)

**Comment by SomeoneYouProbablyKnow** [comment]
30 minutes? Damn, considering fugitives spawn later in the round 30 minutes might more than enough. In any case this is cool, although i wish goof finished his stories PR so that we had a framework for this sort of stuff

File: code/modules/antagonists/fugitive/fugitive_equipment.dm [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+/datum/action/item_action/camouflage/proc/remove_cloaking()
+	do_sparks(2, FALSE, owner)
+	owner.alpha = initial(owner.alpha)
+	to_chat(owner, span_notice("You disable the [name], and become visible once again."))
+	cloaking = FALSE
+
+/obj/item/reagent_containers/hypospray/medipen/invisibility
+	name = "invisibility autoinjector"
+	desc = "An autoinjector containing a stabilized Saturn-X compound. Produced for use in tactical stealth operations, by operatives were presumably comfortable with nudity."
```
Comment (ATH1909) [inline code/modules/antagonists/fugitive/fugitive_equipment.dm:57]:
```suggestion
	desc = "An autoinjector containing a stabilized Saturn-X compound. Produced for use in tactical stealth operations by operatives who were presumably comfortable with nudity."
```

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 4]
```diff
@@ -749,6 +749,13 @@
 		invisible_man.emote("laugh")
 	invisible_man.adjustOrganLoss(ORGAN_SLOT_LIVER, 0.4 * REM * delta_time, required_organtype = affected_organtype)
 
+/datum/reagent/drug/saturnx/stable
+	name = "Stabilized Saturn-X"
+	description = "A chemical extract originating from the Saturn-X compound, stabilized and safer for tactical use. After the recipe was discovered it was planned to be put into mass production, but the program fell apart after its lead disappeared and was never seen again."
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:754]:
```suggestion
	description = "A chemical extract originating from the Saturn-X compound, stabilized and safer for tactical use. After the recipe was discovered, it was planned to be put into mass production, but the program fell apart after its lead disappeared and was never seen again."
```

**Comment by ATH1909** [comment]
this guy should probably get a free storage implant or something to hold his other two pens with, or else he'll be very visible while in SaturnX mode because he has to hold the other two pens for later.

**Comment by Time-Green** [comment]
> this guy should probably get a free storage implant or something to hold his other two pens with, or else he'll be very visible while in SaturnX mode because he has to hold the other two pens for later.

That’s kinda the point though. They’re forced to stash two pens somewhere on the station ahead of time. If they could stay invisible for 30m with no risk, it wouldn’t be much of a fugitive

**Comment by Shadowflame909** [comment]
This is cool. I hope to see fugitives more often

File: code/modules/antagonists/fugitive/fugitive_equipment.dm (hunk 2 above)
Comment (san7890) [reply code/modules/antagonists/fugitive/fugitive_equipment.dm:9]: alright wasn't sure

File: code/modules/antagonists/fugitive/fugitive_equipment.dm [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+/datum/action/item_action/camouflage/Remove(mob/living/remove_from)
+	if(owner)
+		remove_cloaking()
+
+	return ..()
+
+/datum/action/item_action/camouflage/Trigger(trigger_flags)
+	. = ..()
+	if(!.)
+		return
```
Comment (san7890) [inline code/modules/antagonists/fugitive/fugitive_equipment.dm:33]: We should return `FALSE` here instead of null to be nicer to consumers.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Pull request merged**